src/cms/preview-templates/ProjectPreview.js

```diff
@@ -2,20 +2,29 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { ProjectTemplate } from '../../templates/project';
 
-const ProjectPreview = ({ entry, widgetFor }) => (
-  <ProjectTemplate
-    content={widgetFor('body')}
-    description={entry.getIn(['data', 'description'])}
-    tags={entry.getIn(['data', 'tags'])}
-    title={entry.getIn(['data', 'title'])}
-  />
-);
+const ProjectPreview = ({ entry }) => {
+  const entryBaseProducts = entry.getIn(['data', 'relatedproducts']);
+  const baseProducts = entryBaseProducts ? entryBaseProducts.toJS() : [];
+  return (
+    <ProjectTemplate
+      description={entry.getIn(['data', 'description'])}
+      title={entry.getIn(['data', 'title'])}
+      description1={entry.getIn(['data', 'description1'])}
+      title1={entry.getIn(['data', 'title1'])}
```

Comment:
What is the difference between these?

Reply:
It's the first and second description and title.